tests/test_interpreters.py

```diff
@@ -6,24 +6,18 @@
 import py
 import pytest
 
-from tox.config import get_plugin_manager
-from tox.interpreters import ExecFailed
-from tox.interpreters import InterpreterInfo
-from tox.interpreters import Interpreters
-from tox.interpreters import NoInterpreterInfo
-from tox.interpreters import pyinfo
-from tox.interpreters import run_and_get_interpreter_info
-from tox.interpreters import sitepackagesdir
-from tox.interpreters import tox_get_python_executable
+import tox
```

Comment:
this generates a lot of change, and makes generally test lines longer, why move from full imports to this single and then access over the root module?

Reply:
I prefer to way it was before.